rllib/policy/tf_policy.py

```diff
@@ -355,6 +368,25 @@ def get_weights(self):
     def set_weights(self, weights):
         return self._variables.set_weights(weights)
 
+    @override(Policy)
+    def get_state(self):
```

Comment:
Can we add a unit test that checks we restore the optimizer variables correctly?

Reply:
Added.